docs/content/concepts/metadata-tags/asset-metadata.mdx

```diff
@@ -53,12 +58,38 @@ def small_petals(duckdb: DuckDBResource) -> None:
         )
 ```
 
-### Materialization metadata
+### Asset owners
+
+<Note>
+  <strong>Did you know?</strong> If using Dagster+, you can create asset-based
```

Comment:
Owner alerts are only enabled for enterprise customers, if that's worth mentioning

Reply:
It definitely is; thanks for pointing this out!